### PR TITLE
Add 'snakecase', 'camelcase' and 'shuffle'

### PR DESCRIPTION
**Pull request by moul**
I think we can add more functions based on this [xstrings](https://github.com/huandu/xstrings)

### CODE REVIEW

File: functions.go [hunk 1]
```diff
@@ -10,6 +10,7 @@ import (
 	"time"
 
 	util "github.com/aokoli/goutils"
+	"github.com/huandu/xstrings"
```
Comment (technosophos) [inline functions.go]: Since this is imported directly, can you also add it to the `glide.yaml` file? Then I'll get this merged.

**Comment by technosophos** [comment]
Ooo... there are several other functions in xstrings that would be nice to have. Thanks!

File: functions.go (hunk 1 above)
Comment (moul) [inline functions.go]: was already done :)

**Comment by moul** [comment]
Rebased with master

**Comment by technosophos** [comment]
Thanks!

**Pull request merged**